### PR TITLE
Completed Lesson 2 Exercises

### PR DESCRIPTION
**Pull request by TrkCat**
Completed Lesson 2 exercises (and initial break_me.py lesson 1 exercise).

### CODE REVIEW

File: students/TrkCat/Lesson02/drawGrid.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+def gridPart1():
```
Comment (natasha-aleksandrova) [inline students/TrkCat/Lesson02/drawGrid.py]: per pep8 (python style guide): function names and var names should be snake_case -> `grid_part1()`
same for modules (file names) -> `draw_grid.py`

File: students/TrkCat/Lesson02/fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+for num in range(1:101):
+    str = ('')
```
Comment (natasha-aleksandrova) [inline students/TrkCat/Lesson02/fizzbuzz.py]: `str` is a built-in python function which you are overwriting

github highlights built-ins for you (so do some IDEs), so make sure to use alternate var names

File: students/TrkCat/Lesson02/fizzbuzz.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+for num in range(1:101):
```
Comment (natasha-aleksandrova) [inline students/TrkCat/Lesson02/fizzbuzz.py]: `range(1:101)` is an invalid syntax, should be `range(1, 101)`

File: students/TrkCat/Lesson02/fizzbuzz.py (hunk 2 above)
Comment (natasha-aleksandrova) [reply students/TrkCat/Lesson02/fizzbuzz.py]: also, the parens are redundant

**Comment by natasha-aleksandrova** [comment]
@TrkCat make sure to submit github PR link on the course website so that the assignments can get graded.

File: students/TrkCat/Lesson02/fizzbuzz.py (hunk 2 above)
Comment (TrkCat) [reply students/TrkCat/Lesson02/fizzbuzz.py]: Thank you! Definitely don't want to be in the habit of overwriting built-in functions, I'll update the name.

File: students/TrkCat/Lesson02/drawGrid.py (hunk 1 above)
Comment (TrkCat) [reply students/TrkCat/Lesson02/drawGrid.py]: Thanks! It'll take me awhile to break my camel case habits.

File: students/TrkCat/Lesson02/fizzbuzz.py (hunk 3 above)
Comment (TrkCat) [reply students/TrkCat/Lesson02/fizzbuzz.py]: Could have sworn I checked this code through, but looks like I made an (incorrect) update afterwards. Thanks much, will update.

File: students/TrkCat/Lesson02/drawGrid.py (hunk 1 above)
Comment (natasha-aleksandrova) [reply students/TrkCat/Lesson02/drawGrid.py]: understandable, it's challenging to switch between different languages with different style guidelines 🙂

**Pull request merged**